docs/index.html

```diff
@@ -543,14 +541,17 @@ <h3 class="headline-3">illustrations</h3>
 			</p>
 		</div>
 	</section>
+
```

Comment:
das code beispiel kann eingentlich raus. das adden wir dann in die comment section und machen n neues issue auf falls was ist.